cuenca/resources/service_providers.py

```diff
@@ -0,0 +1,13 @@
+from typing import ClassVar
+
+from pydantic.dataclasses import dataclass
+
+from .base import Queryable, Retrievable
+
+
+@dataclass
+class ServiceProvider(Retrievable, Queryable):
+    _resource: ClassVar = 'service_providers'
+
+    name: str
+    provider_key: str
```

Comment:
what's this?

Reply:
Is a "`slug`" to identify the provider, it is used in `omni` and by support to identify the provider and in some cases its kind, i.e: topup or invoice.